tests/Microsoft.Identity.Test.Unit/CoreTests/RegionDiscoveryProviderTests.cs

```diff
@@ -382,44 +400,41 @@ public async Task UpdateApiversionFailsWithNoNewestVersionsAsync()
             Assert.IsTrue(_testRequestContext.ApiEvent.RegionDiscoveryFailureReason.Contains(TestConstants.RegionDiscoveryNotSupportedErrorMessage));
         }
 
-        private void AddMockedResponse(HttpResponseMessage responseMessage, string apiVersion = "2020-06-01", bool expectedParams = true)
+        protected void AddMockedResponse(HttpResponseMessage responseMessage, string apiVersion = "2020-06-01", bool expectedParams = true, bool throwException = false)
```

Comment:
There are already helpers that throw exceptions. Can you just use an existing helper instead of creating a new one?

Reply:
I undid the changes to this function